[2021-03-19 06:15:32,953] {taskinstance.py:826} INFO - Dependencies all met for <TaskInstance: parallel_dag.processing_tasks.bash2 2021-03-18T00:00:00+00:00 [queued]>
[2021-03-19 06:15:33,024] {taskinstance.py:826} INFO - Dependencies all met for <TaskInstance: parallel_dag.processing_tasks.bash2 2021-03-18T00:00:00+00:00 [queued]>
[2021-03-19 06:15:33,025] {taskinstance.py:1017} INFO - 
--------------------------------------------------------------------------------
[2021-03-19 06:15:33,025] {taskinstance.py:1018} INFO - Starting attempt 4 of 4
[2021-03-19 06:15:33,025] {taskinstance.py:1019} INFO - 
--------------------------------------------------------------------------------
[2021-03-19 06:15:33,063] {taskinstance.py:1038} INFO - Executing <Task(BashOperator): processing_tasks.bash2> on 2021-03-18T00:00:00+00:00
[2021-03-19 06:15:33,072] {standard_task_runner.py:51} INFO - Started process 28994 to run task
[2021-03-19 06:15:33,112] {standard_task_runner.py:75} INFO - Running: ['airflow', 'tasks', 'run', 'parallel_dag', 'processing_tasks.bash2', '2021-03-18T00:00:00+00:00', '--job-id', '168', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/parallel_dag.py', '--cfg-path', '/tmp/tmpgqk5itau']
[2021-03-19 06:15:33,113] {standard_task_runner.py:76} INFO - Job 168: Subtask processing_tasks.bash2
[2021-03-19 06:15:33,317] {logging_mixin.py:103} INFO - Running <TaskInstance: parallel_dag.processing_tasks.bash2 2021-03-18T00:00:00+00:00 [running]> on host airflowvm
[2021-03-19 06:15:33,544] {taskinstance.py:1230} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=parallel_dag
AIRFLOW_CTX_TASK_ID=processing_tasks.bash2
AIRFLOW_CTX_EXECUTION_DATE=2021-03-18T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-03-18T00:00:00+00:00
[2021-03-19 06:15:33,545] {bash.py:135} INFO - Tmp dir root location: 
 /tmp
[2021-03-19 06:15:33,545] {bash.py:158} INFO - Running command: 

    echo "2021-03-18"
    echo "2021-03-25"
    echo "hi"

    echo "2021-03-18"
    echo "2021-03-25"
    echo "hi"

    echo "2021-03-18"
    echo "2021-03-25"
    echo "hi"

    echo "2021-03-18"
    echo "2021-03-25"
    echo "hi"

    echo "2021-03-18"
    echo "2021-03-25"
    echo "hi"

[2021-03-19 06:15:33,582] {bash.py:169} INFO - Output:
[2021-03-19 06:15:33,594] {bash.py:173} INFO - 2021-03-18
[2021-03-19 06:15:33,599] {bash.py:173} INFO - 2021-03-25
[2021-03-19 06:15:33,599] {bash.py:173} INFO - hi
[2021-03-19 06:15:33,599] {bash.py:173} INFO - 2021-03-18
[2021-03-19 06:15:33,600] {bash.py:173} INFO - 2021-03-25
[2021-03-19 06:15:33,600] {bash.py:173} INFO - hi
[2021-03-19 06:15:33,600] {bash.py:173} INFO - 2021-03-18
[2021-03-19 06:15:33,600] {bash.py:173} INFO - 2021-03-25
[2021-03-19 06:15:33,600] {bash.py:173} INFO - hi
[2021-03-19 06:15:33,600] {bash.py:173} INFO - 2021-03-18
[2021-03-19 06:15:33,600] {bash.py:173} INFO - 2021-03-25
[2021-03-19 06:15:33,600] {bash.py:173} INFO - hi
[2021-03-19 06:15:33,600] {bash.py:173} INFO - 2021-03-18
[2021-03-19 06:15:33,600] {bash.py:173} INFO - 2021-03-25
[2021-03-19 06:15:33,600] {bash.py:173} INFO - hi
[2021-03-19 06:15:33,601] {bash.py:177} INFO - Command exited with return code 0
[2021-03-19 06:15:33,698] {taskinstance.py:1135} INFO - Marking task as SUCCESS. dag_id=parallel_dag, task_id=processing_tasks.bash2, execution_date=20210318T000000, start_date=20210319T061532, end_date=20210319T061533
[2021-03-19 06:15:33,816] {taskinstance.py:1195} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2021-03-19 06:15:33,839] {local_task_job.py:118} INFO - Task exited with return code 0
